cpu/cortexm_common/mpu.c

```diff
@@ -59,11 +57,17 @@ bool mpu_enabled(void) {
 }
 
 int mpu_configure(uint_fast8_t region, uintptr_t base, uint_fast32_t attr) {
-/* Todo enable MPU support for Cortex-M23/M33 */
-#if __MPU_PRESENT && !defined(CPU_CORE_CORTEX_M23)
+#if __MPU_PRESENT
```

Comment:
```suggestion
/* Todo enable MPU support for Cortex-M23/M33 */
#if __MPU_PRESENT && !defined(__ARM_ARCH_8M_MAIN__) && !defined(__ARM_ARCH_8M_BASE__)
```

If it doesn't work, let's not add code that makes it look as if it might work :wink: